src/index.html

```diff
@@ -1,14 +1,47 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+      <img
+      src="./images/logo.png"
+        alt="logo"
+        class="logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <div><li class="nav__item"><a href="" class="nav__link is-active">APPLE</a></li></div>
+          <li class="nav__item"><a href="" class="nav__link">SAMSUNG </a></li>
```

Comment:
добавляй переносы, не пиши в одну строку 
```suggestion
          <li class="nav__item">
            <a href="" class="nav__link">SAMSUNG </a>
          </li>
```